README.md

```diff
@@ -28,19 +28,13 @@ pip install langkit
 Langkit modules contain UDFs that automatically wire into the collection of UDFs on String features provided by whylogs by default. All we have to do is import the Langkit modules and then instantiate a custom schema as shown in the example below.
 
 ```python 
-from whylogs.experimental.core.metrics.udf_metric import generate_udf_schema
-from whylogs.core.resolvers import STANDARD_RESOLVER
-from whylogs.core.schema import DatasetSchema, DeclarativeSchema
+from whylogs.experimental.core.metrics.udf_metric import udf_metric_schema
 import whylogs as why
 from langkit.sentiment import *
 from langkit.textstat import *
 
-def standard_udf_schema() -> DatasetSchema:
-    return DeclarativeSchema(STANDARD_RESOLVER + generate_udf_schema())
-
-schema=standard_udf_schema()
-
-results = why.log(df, schema=schema)
+text_schema=udf_metric_schema()
```

Comment:
Super minor, but won't the lack of spaces fail a linter (in the future)?

Reply:
fixed